packages/contentstack-bulk-publish/src/commands/cm/bulk-publish/cross-publish.js

```diff
@@ -125,18 +157,76 @@ CrossPublishCommand.flags = {
   retryFailed: flags.string({
     char: 'r',
     description: '(optional) Retry publishing failed entries from the logfile (this flag overrides all other flags)',
+    hidden: true,
+    parse: printFlagDeprecation(['--retryFailed', '-r'], ['--retry-failed']),
+  }),
+  'retry-failed': flags.string({
+    char: 'r',
+    description: '(optional) Retry publishing failed entries from the logfile (this flag overrides all other flags)',
   }),
   bulkPublish: flags.string({
+    char: 'b',
+    hidden: true,
+    description:
+      "This flag is set to true by default. It indicates that contentstack's bulkpublish API will be used to publish the entries",
+    default: 'true',
+    parse: printFlagDeprecation(['--bulkPublish', '-b'], ['--bulk-publish']),
+  }),
+  'bulk-publish': flags.string({
     char: 'b',
     description:
-      "This flag is set to true by default. It indicates that contentstack's bulkpublish API will be used to publish the entries and assets",
+      "This flag is set to true by default. It indicates that contentstack's bulkpublish API will be used to publish the entries",
     default: 'true',
   }),
-  contentType: flags.string({ char: 't', description: 'Content type filter' }),
-  locale: flags.string({ char: 'l', description: 'Locale filter' }),
-  environment: flags.string({ char: 'e', description: 'Source Environment' }),
-  deliveryToken: flags.string({ char: 'x', description: 'Delivery token for source environment' }),
-  destEnv: flags.string({ char: 'd', description: 'Destination Environments', multiple: true }),
+  contentType: flags.string({
+    char: 't',
+    description: 'The Content-Types from which entries need to be published',
+    multiple: true,
+    hidden: true,
+    parse: printFlagDeprecation(['--contentType', '-t'], ['--content-type']),
+  }),
+  'content-type': flags.string({
+    description: 'The Contenttypes from which entries will be published',
+    multiple: true,
+  }),
+  locale: flags.string({
+    hidden: true,
+    char: 'l',
+    description: 'Source locale',
+    parse: printFlagDeprecation(['-l'], ['--locales']),
+  }),
+  locales: flags.string({
+    description: 'Source locale',
+  }),
+  environment: flags.string({
+    char: 'e',
+    description: 'Source Environment',
+    hidden: true,
+    parse: printFlagDeprecation(['--environment', '-e'], ['--source-env']),
+  }),
+  'source-env': flags.string({
+    description: 'Source Env',
+  }),
+  destEnv: flags.string({
+    char: 'd',
+    description: 'Destination Environments',
+    multiple: true,
+    hidden: true,
+    parse: printFlagDeprecation(['--destEnv', '-e'], ['--environment']),
```

Comment:
-e is not configured with destEnv please correct it

Reply:
@netrajpatel I have made the changes. Please check.